ompi/runtime/ompi_mpi_finalize.c

```diff
@@ -205,7 +205,7 @@ int ompi_mpi_finalize(void)
     opal_progress_set_event_flag(OPAL_EVLOOP_ONCE | OPAL_EVLOOP_NONBLOCK);
 #endif
 
-    /* Redo ORTE calling opal_progress_event_users_increment() during
+    /* Redo PRTE calling opal_progress_event_users_increment() during
```

Comment:
I don't believe this will actually work as MPI procs no longer initialize an RTE internally. Thus, there is no point in changing the progress engine to "slow down" the RTE and then recover.

Reply:
All it seems to do is increment a counter internally.

I wouldn't be opposed to removing the call, but I don't know the history there.

Reply:
I don't think we need this any more.  We probably haven't needed it for a long time (but need to clean up the matching code in ompi_mpi_initialize().

Back in the day, there was only one libevent base and it ran in the same thread as the MPI process, and always in fast polling mode.  To reduce MPI latency, if there were no active consumers of the event library (like the TCP BTL or ORTE during init/finalize), we only polled the event library every 100ms or so.

Today, with ORTE having the event library run in its own thread, which is much better, we don't need any of this gunk.  I'm pretty sure we don't need it, anyway.